src/libcrun/cgroup-utils.c

```diff
@@ -406,7 +408,13 @@ destroy_cgroup_path (const char *path, int mode, libcrun_error_t *err)
             {
               ret = rmdir_all (cgroup_path);
               if (ret < 0)
-                repeat = true;
+                {
+                  if (retry_count >= max_attempts)
+                    return crun_make_error (err, errno, "cannot destroy one or subsystem at path `%s`", cgroup_path);
```

Comment:
All the subsystems are under the same hierarchy on cgroup v2.   The error message is "cannot delete path `%s`.

Reply:
this is fixed